common/repo/GitRepo.java

```diff
@@ -381,6 +381,22 @@ public String getHeadCommitId() {
     return commandResult.stdout.trim().replace("\n", "");
   }
 
+  // Retrieves the URLs for a remote (e.g. https://github.com/google/startup-os.git)
+  public String getRemoteURL() {
+    return runCommand("remote get-url origin").stdout;
+  }
+
+  // Returns true if there are commits done since master exist or uncommitted files exist
```

Comment:
Let's rephrase to:
`Checks if there are commits added since master, or any uncommitted files.`

Reply:
Done.